tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestConstSimplifier.scala

```diff
@@ -1,7 +1,8 @@
 package at.forsyte.apalache.tla.pp
 
-import at.forsyte.apalache.tla.lir.TypedPredefs._
-import at.forsyte.apalache.tla.lir.convenience._
+import at.forsyte.apalache.tla.convenience._
+import at.forsyte.apalache.tla.typecomp.{build, TBuilderInstruction}
+import at.forsyte.apalache.tla.lir.convenience.{tla => oldTla}
```

Comment:
what is `oldTla`? We need a better name

Reply:
Yes, but this is only transitional and will be removed soon anyway.